src/main/java/seedu/travelr/model/AddressBook.java

```diff
@@ -100,6 +104,14 @@ public void addTrip(Trip p) {
      */
     public void addEvent(Event e) {
         bucketList.add(e);
+        allEventsList.add(e);
```

Comment:
Actually, should we just have a function called updateAllEvents() that simply does
`allEventsList =  (new UniqueEventList()).addAll(bucketList).addAll(\* All events in trips*\) `

Reply:
Then we can have a function that updates the allEventsList automatically after each operation, instead of us having to modify 2 lists each time